ts/Core/Pointer.ts

```diff
@@ -1540,8 +1555,7 @@ class Pointer {
         selectionMarker[wh] = selectionWH;
         selectionMarker[xy] = selectionXY;
         transform[scaleKey] = scale;
-        transform['translate' + XY] = (transformScale * plotLeftTop) +
-            (touch0Now - (transformScale * touch0Start));
+        transform['translate' + XY] = (touch0Now - (transformScale * touch0Start));
```

Comment:
Why removed `(transformScale * plotLeftTop) +`? I always ask when some part of code is removed, because there probably is a reason why that code was added in the first time, right?

Reply:
we want to seperatly calculate this values for each series, so this part was moved to `scaleGroups` function. plotLeft/plotTop is different for each series, depending on the axis, it is added to, so it should be calculated separately.